### ISSUE TITLE
NullPointerException: (TgBot.kt)

### ISSUE THREAD

**Issue by JellyBrick** [kraftwerk28/spigot-tg-bridge#19]
```
Caused by: java.lang.NullPointerException
[16:03:30 WARN]:        at org.kraftwerk28.spigot_tg_bridge.TgBot.onTextHandler(TgBot.kt:190)
[16:03:30 WARN]:        at org.kraftwerk28.spigot_tg_bridge.TgBot.handleUpdate(TgBot.kt:108)
[16:03:30 WARN]:        at org.kraftwerk28.spigot_tg_bridge.TgBot$initHandler$1.invokeSuspend(TgBot.kt:97)
[16:03:30 WARN]:        at kotlin.coroutines.jvm.internal.BaseContinuationImpl.resumeWith(ContinuationImpl.kt:33)
[16:03:30 WARN]:        at kotlinx.coroutines.DispatchedTask.run(DispatchedTask.kt:106)
[16:03:30 WARN]:        at kotlinx.coroutines.scheduling.CoroutineScheduler.runSafely(CoroutineScheduler.kt:571)
[16:03:30 WARN]:        at kotlinx.coroutines.scheduling.CoroutineScheduler$Worker.executeTask(CoroutineScheduler.kt:750)
[16:03:30 WARN]:        at kotlinx.coroutines.scheduling.CoroutineScheduler$Worker.runWorker(CoroutineScheduler.kt:678)
[16:03:30 WARN]:        at kotlinx.coroutines.scheduling.CoroutineScheduler$Worker.run(CoroutineScheduler.kt:665)
[16:03:30 WARN]: kotlinx.coroutines.JobCancellationException: Parent job is Cancelling; job=JobImpl{Cancelling}@6f32d593
```

**Comment by kraftwerk28** [comment]
Thanks for the issue! Which version of plugin are you using?

**Comment by kraftwerk28** [comment]
Ah, my bad, it should be fixed in d819420811ceadbfef7ef675551fa8a631f0264d and in release 0.17
